test/app-tap/gh-5040-inter-mode-isatty-via-errinj.test.lua

```diff
@@ -12,7 +12,7 @@ local fio = require('fio')
 --
 
 local TARANTOOL_PATH = arg[-1]
-local output_file = fio.abspath('out.txt')
+local output_file = fio.abspath('gh-5040_out.txt')
```

Comment:
Can we meet the same problem between test cases?

Reply:
> Can we meet the same problem between test cases?

Now, I don't see the reasons for that. But I've changed file names to distinguish outputs of these tests better.

Reply:
The root of the problem that you're fixed here is presence of the output file when a test case started, right? My question is whether a second test case within given test may experience a problem of this kind. The test cases use the same output file name.

Reply:
> The root of the problem that you're fixed here is presence of the output file when a test case started, right? My question is whether a second test case within given test may experience a problem of this kind. The test cases use the same output file name.

If both test cases use the same file name for output file and one test case clear output file after itself ran, failure in second case shouldn't appear. I added different names for output files for another reason, should I do it in another commit?

Reply:
I see now: `os.remove(output_file)` is within the loop by test cases. Sorry, nevermind.

Reply:
> should I do it in another commit?

I don't think it worthful.